effekt/shared/src/main/scala/effekt/RecursiveDescent.scala

```diff
@@ -674,20 +674,23 @@ class RecursiveDescent(positions: Positions, tokens: Seq[Token], source: Source)
     nonterminal:
       Region(`region` ~> idDef(), stmt())
 
-  def boxExpr(): Term =
+  def boxExpr(): Term = {
     nonterminal:
-      val captures = `box` ~> backtrack(captureSet())
+      consume(`box`)
       val expr = if (peek(`{`)) functionArg()
-        else if (peek(`new`)) newExpr()
-        else callExpr()
-      Box(captures.unspan, expr)
-
+      else if (peek(`new`)) newExpr()
```

Comment:
Do we need this case?
I think it was there before because of some shift-reduce conflict, but every `newExpr` is a `callExpr`? [?]